Makefile

```diff
@@ -37,6 +37,8 @@ check_license:
 	@./scripts/check_license.sh
 
 test:
+	@echo ">> installing consul binary used in tests"
+	@$(GO) get -u github.com/hashicorp/consul
```

Comment:
Yeah, maybe now. It still needs the consul binary, I'm not sure how to get this with govendor.